File: randoms/randoms.java
import java.util.Random;

public class Randoms {

    public static void main(String[] args){
        Random rand = new Random();
        int[] listOfRandomNumbers = new int[50];
        for (int i = 0; i < 50; i++) {
            listOfRandomNumbers[i] = rand.nextInt(100) + 1;
        }
        
        // This printed string is "String Data File" for the Test Suite input.
        String listOfNumbers = "";
        for(int index : listOfRandomNumbers) {
            listOfNumbers += index;
        }
        System.out.println(listOfNumbers);
    }
}

File: randoms/randoms.js
let listOfRandomNumbers = [];
for (let i = 0; i < 50; i++) {
  listOfRandomNumbers.push(Math.floor(Math.random() * 100) + 1);
}

let listOfNumbers = "";
for(var i = 0; i < listOfRandomNumbers.length; i++) {
    listOfNumbers += i;
}
console.log(listOfNumbers);
